src/analyses/variable-sensitivity/three_way_merge_abstract_interpreter.cpp

```diff
@@ -125,9 +125,9 @@ bool ai_three_way_merget::visit_edge_function_call(
     // but in VSD it does, so this cannot be omitted.
     s_working.transform(
       callee_function_id,
-      l_callee_end,
+      p_callee_end,
       calling_function_id,
-      l_return_site,
+      p_return_site,
```

Comment:
Is "p" short for "trace pointer?" I think "l" I could see as "location," but this one seemed less clear. Saying this while I'm aware that this isn't really related to this PR.

Reply:
Yes and maybe `t_` would be better.  Having similar names makes it much more readable IMHO.